src/Form/RadioButtons.php

```diff
@@ -4,14 +4,32 @@
 
 namespace MyParcelCom\JsonSchema\FormBuilder\Form;
 
-use MyParcelCom\JsonSchema\FormBuilder\Properties\Meta\MetaFieldType;
+use MyParcelCom\JsonSchema\FormBuilder\SchemaProperties\Meta\MetaFieldType;
+use MyParcelCom\JsonSchema\FormBuilder\Translations\LabelTranslationCollection;
 use Override;
 
 class RadioButtons extends ChoiceField
 {
+    public function __construct(
+        string $name,
+        string $label,
+        OptionCollection $options,
+        bool $isRequired = false,
+        ?string $help = null,
+        ?LabelTranslationCollection $labelTranslations = null,
+        public ?string $initialValue = null,
```

Comment:
Here we have the value as public, but previously as protected. Is it just a miss? 
```suggestion
        private readonly ?string $initialValue = null,
```

Reply:
Yeah I missed that one. I'll make all `private readonly`